code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -598,7 +598,12 @@
 		delay = 1 SECONDS
 		playsound(loc, "alien_doorpry", 25, TRUE)
 	else
-		delay = 4 SECONDS
+		if(M.mob_size == MOB_SIZE_XENO_SMALL && MOB_SIZE_XENO_SMALL && MOB_SIZE_XENO_VERY_SMALL)
```

Comment:
Probably should’ve made this two comments to begin with.

This needs fixing, you could do a switch statement if you’re fancy.
You might also want to add a default value